README.md

```diff
@@ -49,19 +54,19 @@ Let's start our server by using `npm start`. Visit http://localhost:8080 in your
 import { createStore } from 'redux';
 import counterReducer from './reducers/counterReducer';
```

Comment:
This section is to 'play around in console' but you cannot use `import` in the console. It seems we want to tell people to add these `console.log()` and `store.dispatch()` calls to `index.js` instead and refresh the page while watching the console.